modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-mode.adoc

```diff
@@ -21,6 +21,10 @@ Production (prod) mode disables dev-mode settings:
 
 It also enables a set of tuners of the autotuner. For descriptions about the tuners, see xref:./rpk-redpanda-tune-list.adoc#tuners[Tuners] in the xref:./rpk-redpanda-tune-list.adoc[rpk redpanda tune list] command reference.
 
+== Recovery mode
+
+Recovery mode sets the broker configuration property `recovery_mode_enabled` to `true`. This provides a stable environment for troubleshooting and restoring a failed cluster. See xref:manage:recovery-mode.adoc[] for more information.
```

Comment:
Something's nt right here--please place text in brackets rather than showing the link tag. Also--take a look in the Flags table. Some of the items are not tagged correctly. Probably missed by the script.